src/CalcViewModel/UnitConverterViewModel.cpp

```diff
@@ -782,13 +782,16 @@ void UnitConverterViewModel::RefreshSupplementaryResults()
 // values are not both zero.
 void UnitConverterViewModel::AnnounceConversionResult()
 {
-    if ((m_valueFromUnlocalized != m_lastAnnouncedFrom || m_valueToUnlocalized != m_lastAnnouncedTo) && Unit1 != nullptr && Unit2 != nullptr)
+    if ((m_valueFromUnlocalized != m_lastAnnouncedFrom
+        || m_valueToUnlocalized != m_lastAnnouncedTo)
+        && m_Unit1 != nullptr
+        && m_Unit2 != nullptr)
     {
         m_lastAnnouncedFrom = m_valueFromUnlocalized;
         m_lastAnnouncedTo = m_valueToUnlocalized;
 
-        Unit ^ unitFrom = Value1Active ? Unit1 : Unit2;
-        Unit ^ unitTo = (unitFrom == Unit1) ? Unit2 : Unit1;
+        Unit^ unitFrom = Value1Active ? m_Unit1 : m_Unit2;
```

Comment:
nit: Could be out of scope but m_Unit1 and m_Unit2 don't feel like very descriptive variable names. New to the codebase so could be wrong though.